actionmailbox/test/dummy/config/environments/development.rb

```diff
@@ -30,7 +30,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :local
+  config.active_storage.services = [:local]
```

Comment:
This is a breaking change that will need a deprecation cycle. We need to support `config.active_storage.service` in 6.1 so people have a chance to upgrade.

Reply:
I've changed it so that `config.active_storage.service` continues to work.